src/App.jsx

```diff
@@ -27,16 +30,41 @@ class App extends Component {
         this.state = {
             text: "",
             destination: "vatriume",
-            notify: false
+            notify: false,
+            whiteTheme: false
```

Comment:
Please add trailing comma, to avoid this kind of diffs in the future.

Reply:
> Please add trailing comma, to avoid this kind of diffs in the future.

Done.